ngx_rtmp_relay_module.c

```diff
@@ -218,7 +246,7 @@ ngx_rtmp_relay_static_pull_reconnect(ngx_event_t *ev)
                    "relay: reconnecting static pull");
 
     ctx = ngx_rtmp_relay_create_connection(&rs->cctx, &rs->target->name,
-                                           rs->target);
+                                           rs->target, NULL);
```

Comment:
No need to add an extra parameter.
For a static pull, if there is  a variables in url, give 'variables not supported in static pull' information when Nginx parses conf file and quit.

Reply:
Static pull will not support this substitution. This extra parameter is purely because this function is used in ngx_rtmp_relay_static_pull_reconnect and we need to pass the session in.

Reply:
I insist that there is no need to add an extra parameter.

Reply:
it is related to the last review